mParticle-AppsFlyer/MPKitAppsFlyer.m

```diff
@@ -233,7 +237,7 @@ - (nonnull MPKitExecStatus *)routeCommerceEvent:(nonnull MPCommerceEvent *)comme
         action == MPCommerceEventActionCheckout ||
         action == MPCommerceEventActionPurchase)
     {
-        NSMutableDictionary *values = [NSMutableDictionary dictionary];
+        NSMutableDictionary *values = [commerceEvent.customAttributes mutableCopy];
```

Comment:
Seems like this could result in an empty dictionary if no customAttributes were set. Maybe something like `= commerceEvent.customAttributes ? [commerceEvent.customAttributes mutableCopy] : [NSMutableDictionary dictionary]`